include/eigenpy/std-vector.hpp

```diff
@@ -352,6 +352,44 @@ struct contains_vector_derived_policies
     return contains_algo<key_type>::run(container, key);
   }
 };
+
+///
+/// \brief Add standard method to a std::vector.
+/// \tparam NoProxy When set to false, the elements will be copied when
+/// returned to Python.
+///
+template <typename Container, bool NoProxy, typename CoVisitor>
+struct AddStdMethodToStdVector
```

Comment:
```suggestion
struct ExposeStdMethodToStdVector
```
might be more exprisse.